python/py-sure/Portfile

```diff
@@ -19,31 +19,34 @@ homepage            http://github.com/gabrielfalcao/sure
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
```

Comment:
`master_sites` and `distname` are set by the `python` PG so you can remove these lines